client/assets/stylesheets/gutenboarding.scss

```diff
@@ -0,0 +1,23 @@
+// External Dependencies
```

Comment:
Should this file live here? Or should we colocate it more with the code it is relevant for (i.e. move it to `client/landing/gutenboarding/`), in the spirit of breaking `client/assets/stylesheets/` down into separate units as much as possible (to separate concerns and make clear what is used for what)?

Reply:
I agree entry-points shouldn't be in `client/assets/stylesheets/*`. 👍

Reply:
@jsnajdr did you have thoughts on entry-file locations?